Shopify/cli
# Remove automatic enable/disable dev preview when running dev and deploy

### WHY are these changes introduced?

Partially fixes https://github.com/Shopify/develop-app-foundations/issues/167



### WHAT is this pull request doing?
Revert the changes of this [PR](https://github.com/Shopify/cli/pull/2171) 


### How to test your changes?
Run `dev` and `deploy` with `unified` enabled:
- You should not see the messages `enabled preview message` and `disabled preview message`
- If you access the app's extension section in the partners dashboard you can check that the dev preview state is not modified


### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
